doc/rados/api/objclass-sdk.rst

```diff
@@ -2,6 +2,8 @@
 SDK for Ceph Object Classes
 ===========================
 
+.. _`rados-objclass-api-sdk`:
```

Comment:
might want to put the label *before* the section header. something like
```rst
.. _`rados-objclass-api-sdk`:

SDK for Ceph Object Classes
===========================
```

Reply:
see https://ceph--40152.org.readthedocs.build/en/40152/dev/osd-class-path/ for the rendered page.

Reply:
uh kay sorry :)

Reply:
thank you! it's perfect now!